### ISSUE TITLE
Checkboxes and other control elements are tiny on 4k screens

### ISSUE THREAD

**Issue by kblcuk** [ChrisRM/material-theme-jetbrains#1260]
### Summary
After recent update checkboxes and other control elements became tiny.
...

### Expected behavior
Look with default Intellij theme:
![image](https://user-images.githubusercontent.com/5792253/59756237-00ee2b80-9292-11e9-8545-8c4181424692.png)

...

### Actual behavior

![image](https://user-images.githubusercontent.com/5792253/59756127-cdab9c80-9291-11e9-91e9-dbdf485c6128.png)

Feels like it generally broken, since install screen already has tiny checkboxes: 
![image](https://user-images.githubusercontent.com/5792253/59757843-55df7100-9295-11e9-96b2-679a5c0f5b99.png)

...

### Environment

WebStorm 2019.1.3
Build #WS-191.7479.14, built on May 27, 2019

JRE: 1.8.0_202-release-1483-b53 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 5.0.0-15-generic

Plugin version: latest (v3.10)

`material_theme.xml`:
```
<application>    
  <component name="MaterialThemeConfig">    
    <option name="allowDataCollection" value="true" />    
    <option name="compactDropdowns" value="true" />    
    <option name="compactSidebar" value="true" />    
    <option name="darkTitleBar" value="true" />    
    <option name="isCompactMenus" value="true" />    
    <option name="isCompactStatusBar" value="true" />    
    <option name="isCompactTables" value="true" />    
    <option name="isDecoratedFolders" value="false" />    
    <option name="isWizardShown" value="true" />    
    <option name="overrideAccentColor" value="true" />    
    <option name="pristineConfig" value="false" />    
    <option name="statusBarTheme" value="false" />    
    <option name="upperCaseButtons" value="false" />    
    <option name="settingsSelectedTab" value="2" />    
    <option name="accentColor" value="009688" />    
    <option name="selectedTheme" value="CUSTOM" />    
    <option name="userId" value="7ad19a:167b762deca:-8000" />    
    <option name="version" value="3.10.0" />    
  </component>    
</application>    
```
`material_custom_theme.xml`:
```
<application>    
  <component name="MaterialCustomThemeConfig">    
    <option name="excludedColor" value="2e3c43ff" />    
    <option name="accentColor" value="009688ff" />    
    <option name="notificationsColor" value="323232ff" />    
    <option name="treeSelectionColor" value="546e7aff" />    
    <option name="highlightColor" value="425b67ff" />    
    <option name="secondBorderColor" value="2a373eff" />    
    <option name="tableSelectedColor" value="314549ff" />    
    <option name="contrastColor" value="1e272cff" />    
    <option name="disabledColor" value="415967ff" />    
    <option name="secondaryBackgroundColor" value="32424aff" />    
    <option name="buttonColor" value="2e3c43ff" />    
    <option name="selectionForegroundColor" value="ffffffff" />    
    <option name="selectionBackgroundColor" value="546e7aff" />    
    <option name="textColor" value="607d8bff" />    
    <option name="foregroundColor" value="b0bec5ff" />    
    <option name="backgroundColor" value="263238ff" />    
  </component>    
</application>   
```

Thanks!

**Comment by paesku** [comment]
I can confirm this on latest Ubuntu as well. 
```
WebStorm 2019.1.3
Build #WS-191.7479.14, built on May 27, 2019

JRE: 1.8.0_202-release-1483-b53 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.18.0-22-generic
```

**Comment by AlexDaSoul** [comment]
PhpStorm 2019.1.3
Build #PS-191.7479.51, built on June 18, 2019

JRE: 1.8.0_202-release-1483-b53 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 5.0.0-19-generic

![Screenshot from 2019-06-28 15-33-09](https://user-images.githubusercontent.com/1850022/60343176-13681380-99bc-11e9-9783-a2bd2e571b64.png)

**Comment by mallowigi** [comment]
Looks like the Material Components feature is disabled. Can you enable it and try again? Thanks.

**Comment by AlexDaSoul** [comment]
It was enabled. But I turned off material components, and checkboxes became normal.

**Comment by kblcuk** [comment]
Same as what @AlexDaSoul said, it was enabled.
While disabling it brings checkboxes to normal state, I can't really say that the whole look works without them enabled:
![image](https://user-images.githubusercontent.com/5792253/61202366-ff2c5200-a6ef-11e9-8be2-30d91243f59d.png)

Enabling them back afterwards brings everything to look showed in original message.
@mallowigi can you explain why you decided that "Material components" are disabled?

**Comment by kblcuk** [comment]
Well, whatever it was, it got fixed by 2019.2.
@mallowigi I still think closing issue before getting user feedback is good way to handle issues :/
